### PR TITLE
Play nicely with YUI Compressor

### PR DESCRIPTION
**Pull request by envoked**
I ran into an issue in using brace and yui compressor which was related to there being a variable named `char`. Figured this might help others who might run into similar issues.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3730,11 +3730,11 @@ var MouseHandler = function(editor) {
...
-        var char = editor.renderer.screenToTextCoordinates(e.x, e.y);
+        var character = editor.renderer.screenToTextCoordinates(e.x, e.y);
         var range = editor.session.selection.getRange();
         var renderer = editor.renderer;
 
-        if (!range.isEmpty() && range.insideStart(char.row, char.column)) {
+        if (!range.isEmpty() && range.insideStart(character.row, character.column)) {
             renderer.setCursorStyle("default");
         } else {
             renderer.setCursorStyle("");
```
Comment (mstorus) [inline index.js]: FYI that change was merged into ACE here https://github.com/ajaxorg/ace/pull/2275

**Comment by thlorenz** [comment]
Could we please update ACE instead since this was fixed there?
See instructions in the readme WRT to how to updated and regenerate.
Thanks.

**Comment by mstorus** [comment]
Blocked on ACE creating a new version tag (1.1.9)

**Comment by thlorenz** [comment]
Not sure what you mean, just update the tag [here](https://github.com/thlorenz/brace/blob/master/build/update.js#L22) and rerun the update script.

**Comment by thlorenz** [comment]
Ah, sorry, got it - they didn't create a tag.
In the worst case specify a commit hash for now (that should work as well).
Make sure to comment why though.

**Pull request closed without merging**